spec/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(author_id: 1, title: 'Tanks', comments_counter: 0, likes_counter: 0) }
+
+  before { subject.save }
+
+  context 'when title is not present' do
+    before { subject.title = nil }
+
+    it 'is not valid' do
+      expect(subject).not_to be_valid
+    end
+  end
+
+  context 'when comments_counter is not a number' do
+    before { subject.comments_counter = 'one' }
+
+    it 'is not valid' do
+      expect(subject).not_to be_valid
+    end
+  end
+
+  context 'when comments_counter is less than 0' do
+    before { subject.comments_counter = -1 }
+
+    it 'is not valid' do
+      expect(subject).not_to be_valid
+    end
+  end
+end
```

Comment:
- [ ] You did a great job working on your tests! 💯 However, according to the requirements:

![image](https://github.com/IsmailChangezi/blog-app/assets/77069011/2074651e-8b1e-4221-9113-9040499773b8)

Please make sure to include tests for all methods and validations. Currently, there aren't any tests for: `Title must not exceed 250 characters.`, `LikesCounter must be an integer greater than or equal to zero.`, `Post#most_recent_comments`, `Post#post_counter_update`. Please make sure to add them! 🤓 🔧